sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/alibaba.yaml

```diff
@@ -0,0 +1,84 @@
+        link: https://owasp.org/Top10/A07_2021-Identification_and_Authentication_Failures
+        type: Standards
+      - description: OWASP Top 10 2017 Category A3 - Sensitive Data Exposure
+        link: https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
+        type: Standards
+      - description: MITRE, CWE-798 - Use of Hard-coded Credentials
+        link: https://www.owasp.org/www-project-top-ten/2017/A3_2017-Sensitive_Data_Exposure
+        type: Standards
+      - description: MITRE, CWE-259 - Use of Hard-coded Password
+        link: https://cwe.mitre.org/data/definitions/259
+        type: Standards
+      - description: SANS Top 25 - Porous Defenses
+        link: https://www.sans.org/top25-software-errors/#cat3
+        type: Standards
+  detection:
+    post:
+      statisticalFilter: 
+        threshold: 4.2
+      patternNot: "EXAMPLE"
+  rules:
+    - id: S6336
+      metadata:
+        name: Alibaba Secret Access Keys should not be disclosed
+        message: Make sure this Alibaba Cloud Access Key ID gets revoked, changed, and removed from the code.
+        impact: alibaba-secret-access.adoc
+      examples:
+        - text: >
+            String fileContent = "LTAI5tBcc9SecYAo";
+          containsSecret: true
+          match: LTAI5tBcc9SecYAo
+        - text: >
+            String fileContent = "LTAI5tBcc9SecYAomgyUSFs8";
+          containsSecret: true
+          match: LTAI5tBcc9SecYAomgyUSFs8
+        - text: >
+            LNTTAI5tBcc9SecYAomgyUSFs8
+            // extra characters before
+            assertThat(analyze(check, "BEFLNTTAI5tBcc9SecYAomgyUSFs8")).isEmpty();
+            // extra characters after
+            assertThat(analyze(check, "LNTTAI5tBcc9SecYAomgyUSFs8AFT")).isEmpty();
+          containsSecret: false
+      detection:
+        matching:
+          pattern: "\\b(LTAI[0-9A-Za-z]{12}(:?[0-9A-Za-z]{8})?)\\b"
+
+    - id: S6336
+      metadata:
+        name: Alibaba Access Key Secrets should not be disclosed
+        message: Make sure this Alibaba Cloud Access Key Secret gets revoked, changed, and removed from the code.
+        impact: alibaba-secret-access.adoc
+      examples:
+        - text: >
+            String fileContent = String aliyunAccessKeySecret="KmkwlDrPBC68bgvZiNtrjonKIYmVT8";
+          containsSecret: true
+          match: KmkwlDrPBC68bgvZiNtrjonKIYmVT8
+        - text: >
+            String aliyunAccessKeySecret = "KmkwlDrPBC68bgvZiNtrjonKIYmVT8";
+          containsSecret: true
+          match: KmkwlDrPBC68bgvZiNtrjonKIYmVT8 
+        - text: >
+            String aliyunAccessKeySecret = "100000000000000000000000000000";
+          containsSecret: false
+      detection:
+        matching:
+          pattern: "(?i)(?<![A-Z])ali(?:yun|baba|cloud).{0,50}['\"`]([0-9a-z]{30})['\"`]"
+    - id: S6336
+      metadata:
+        name: Alibaba Access Key IDs should not be disclosed
```

Comment:
It says "Access Key IDs" here, but this seems to be about "Access Key Secrets".

Reply:
```suggestion
        name: Alibaba Access Key Secrets
```